lib/adal.js

```diff
@@ -662,15 +716,33 @@ var AuthenticationContext = (function () {
 
     /**
      * Gets login error
-     * @returns {string} error message related to login
+     * @returns {string} error message related to login.
      */
     AuthenticationContext.prototype.getLoginError = function () {
         return this._getItem(this.CONSTANTS.STORAGE.LOGIN_ERROR);
     };
 
     /**
-     * Gets requestInfo from given hash.
-     * @returns {string} error message related to login
+     * Parameters object created from the response received from AAD.
+     *  @class Parameters
```

Comment:
I don't think this qualifies for a class. This is the list of query parameters passed in the response from AAD. We cannot really define the interface for this as it can change based on query parameters in the response.